README.md

```diff
@@ -12,62 +12,361 @@ Install from NPM:
 npm install @brightspace-ui/testing
 ```
 
-## Developing, Testing and Contributing
+## Writing Tests
 
-After cloning the repo, run `npm install` to install dependencies.
+Tests leverage the familiar [Mocha](https://mochajs.org/) and [Chai](https://www.chaijs.com/) test frameworks. Many of the available utilities are wrappers around or extensions of the excellent [@open-wc testing helpers](https://open-wc.org/docs/testing/helpers/).
 
-### Linting
+### describe, it, assert & expect
 
-```shell
-# eslint
-npm run lint
+Tests can be grouped into suites using `describe` and are defined inside `it` blocks.
+
+Results can be verified using either the BDD-style `expect` or TDD-style `assert` syntax (although try not to mix & match).
```

Comment:
> (although try not to mix & match)

Lol and then you do just that. Maybe broken into two different code blocks?
```javascript
import { expect } from '@brightspace-ui/testing';

describe('group of tests', () => {
  it('should multiply numbers', () => {
    expect(2 * 4).to.equal(8);
  });
});
```
```javascript
import { assert} from '@brightspace-ui/testing';

describe('group of tests', () => {
  it('should multiply numbers', () => {
    assert.equal(2 * 4, 8);
  });
});
```

Maybe that's overkill